app/views/health/patient/_engagement_item.haml

```diff
@@ -1,6 +1,7 @@
 %li
   - status, label, url, modal, tooltip = form_status
-  = if_link_to(link, url, data: { loads_in_pjax_modal: modal }) do
+  - data = {data: { loads_in_pjax_modal: true }} if modal
+  = if_link_to(link, url, data) do
```

Comment:
Isn't it going to complain that data isn't defined if `modal` is false?

Reply:
doesn't seem to